docs/root/development/performance/device_conditions.rst

```diff
@@ -30,7 +33,45 @@ Experimentation method
 Modified versions of the "hello world" example apps were used to run the following experiments.
 See the iOS/Android sections below for instructions on building the examples.
 
-Throughout all of the following steps, we tested to make sure that network requests succeeded
+iOS
+---
+
+The original investigation was completed as part of :issue:`this issue <128#issuecomment-516260951>`.
+
+We built the Envoy Mobile library using the following flags to allow us to build to
+a device with debugging symbols:
+
+``bazel build ios_dist --config=ios --ios_multi_cpus=arm64 --copt=-ggdb3``
+
+In the active scheme of the app's ``Environment Variables``, set ``CFNETWORK_DIAGNOSTICS=3``
+to enable more verbose ``CFNetwork`` logs. Additionally, set Envoy's logs to ``trace``.
+
+Within the app, we performed the following:
+
+- Made single requests to a simple Python server from which we could see when the client connected, disconnected, and made requests
```

Comment:
Are you able to link the python server code?

Reply:
Sure